src/formio/components/index.ts

```diff
@@ -1,11 +1,14 @@
 // Input components
+export * from './checkbox';
 export * from './textfield';
 export * from './email';
 export * from './date';
 export * from './datetime';
 export * from './time';
 export * from './phonenumber';
 export * from './postcode';
+export * from './radio';
+export * from './selectboxes';
 export * from './number';
 export * from './file';
```

Comment:
these exports should also be ordered according to the ordering in the form builder :)